ansible/roles/debops.bootstrap/defaults/main.yml

```diff
@@ -35,6 +35,13 @@ bootstrap__domain: '{{ inventory_hostname.split(".")[1:] | join(".") }}'
 # Set custom DNS hostname on a given host.
 bootstrap__hostname: '{{ inventory_hostname_short | d(inventory_hostname.split(".")[0]) }}'
 
+                                                                   # ]]]
+# .. envvar:: bootstrap__etc_hosts [[[
+#
+# Add the hostname to /etc/hosts and make it point to 127.0.1.1
+# This is usually what you want except if you manage yourself the
+# hostname entry and want it to point to some other IP.
+bootstrap__etc_hosts: True
```

Comment:
Can you add an empty line after the variable? This will make the variable definitions uniform.

Reply:
Added.